exercises/acronym/examples/src/Acronym.purs

```diff
@@ -0,0 +1,48 @@
+module Acronym
+  ( abbreviate
+  ) where
+
+import Prelude
+import Data.String as String
+import Data.Char as Char
+import Data.List as List
+import Data.List (List(Nil), (:))
+import Data.Array as Array
+import Data.Char.Unicode as UChar
+
+abbreviate :: String -> String
+abbreviate str =
+  String.fromCharArray
+  $ Array.fromFoldable
+  $ initials
+  $ List.fromFoldable
```

Comment:
Also is there a way to write this so that it reads from top to bottom? Like with Elixir/Elm/F#s pipe operator?

```elm
value
|> firstFunc
|> secondFunc
|> thirdFunc
```